scipy/signal/splinemodule.c

```diff
@@ -551,7 +551,7 @@ static struct PyModuleDef moduledef = {
     NULL
 };
 
-PyObject *PyInit_spline(void)
+PyObject *PyInit__spline(void)
```

Comment:
Minor point, but on line 544 the module name should be changed to `"scipy.signal._spline"`.